cdap-proto/src/main/java/co/cask/cdap/proto/metadata/lineage/ProgramFieldOperationInfo.java

```diff
@@ -16,14 +16,20 @@
 
 package co.cask.cdap.proto.metadata.lineage;
 
+import co.cask.cdap.api.annotation.Beta;
+
 import java.util.ArrayList;
 import java.util.Collections;
 import java.util.List;
 
 /**
- * Represents information about the field operations along with the programs who
- * performed them.
+ * In field lineage details, each incoming or outgoing record for a particular
```

Comment:
In field lineage details, represents a set of programs that performed the same operations.

Reply:
fixed.